Gems/PhysX/Code/Editor/EditorJointConfiguration.h

```diff
@@ -146,6 +146,8 @@ namespace PhysX
 
         float m_forceMax = 1.0f;
         float m_torqueMax = 1.0f;
+        
+        AZStd::string m_jointName{""};
```

Comment:
nit: An `AZStd::string` has a default constructor and will be initialized to a known state (empty string) so the `{""}` is superfluous here and could be removed.